<launch>
    <param name="robot_description" command="xacro '$(find nuturtle_description)/urdf/turtlebot3_burger.urdf.xacro'"/>
    <rosparam command="load" file="$(find nuturtle_description)/config/diff_params.yaml"/>
    <rosparam command="load" file="$(find rigid2d)/config/frame_id_params.yaml"/>
    <!-- <param name="odom_frame_id" type="string" value="odom"/>
    <param name="body_frame_id" type="string" value="base_footprint"/>
    <param name="left_wheel_joint" type="string" value="wheel_left_joint"/>
    <param name="right_wheel_joint" type="string" value="wheel_right_joint"/> -->

    <node name="turtlebot3_teleop_keyboard" pkg="turtlebot3_teleop" type="turtlebot3_teleop_key" output="screen"/>
    <node name="odometer" pkg="rigid2d" type="odometer" output="screen" />
    <node name="fake_turtle" pkg="rigid2d" type="fake_turtle"/>
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
    <node name="rviz" pkg="rviz" args="-d $(find rigid2d)/config/turtle_model.rviz -f odom" type="rviz"/>

</launch>